crates/sui-indexer/src/handlers/committer.rs

```diff
@@ -44,15 +41,6 @@ pub async fn start_tx_checkpoint_commit_task<S>(
         if indexed_checkpoint_batch.is_empty() {
             continue;
         }
-        if config.skip_db_commit {
```

Comment:
What is this one about?

Reply:
this was added a while ago to benchmark perf w/ and w/o db commits, but it turned out that the DB commit was the sole bottleneck, so no longer need this config any more.